CHANGELOG.md

```diff
@@ -122,12 +189,16 @@ These functions and structs can be referenced in both code and blueprints and it
 - Fixed client connection not being cleaned up when moving out of interest of a server.
 - Fixed an assertion being triggered on async loaded entities due to queuing some component addition.
 - Fixed a bug where consecutive invocations of CookAndGenerateSchemaCommandlet for different levels could fail when running the schema compiler.
+- Fixed an issue where GameMode values won't be replicated between server workers if it's outside their Interest.
```

Comment:
Likewise - were these bugs fixed in 0.13 but added to the 0.12 block mistakenly?

Reply:
Thanks for spotting. It took ages but it seems the only one which is in the wrong place is 
`- Fixed an issue where a NetworkFailure won't be reported when connecting to a deployment that doesn't support dev_login with a developer token, and in some other configuration-dependent cases.` 
Rest are extra notes added in the 0.12-rc-cleanup merge back into master, so those commits never made it into release.